### PR TITLE
Aesthetic, functional improvements for Graphical Display

### PR DESCRIPTION
**Pull request by thinkyhead**
I just got a Graphical Display, so of course the first thing I did was fix some display issues and improve the aesthetic a little bit. See if this agrees with your taste and preferences.

One bug / issue is fixed. X and Y can now show negative values. The decimal point is sacrificed for all cases. It could be revived for positive values. (It would be good to consolidate the number-to-string routines a little at the same time.)
- Fixes a bug in `language_an.h` — `MSG_E` was incorrectly set to `"y"`.
- Nozzle / Bed Temperatures are now centered with their graphic.
- Display X, Y, Z prefixes in uppercase
- Don't display a faux colon after the X, Y, Z labels
- XY are now displayed in the same style as character displays:
  - `___4` - 1 digit
  - `__25` - 2 digits
  - `_172` - 3 digits
  - `__-5` - 1 digit, negative
  - `_-99` - 2 digit, negative
  - `-999` - 3 digit, negative
- Display a blank space instead of `--:--` before the print timer is set
- Display a blank space instead of `---` when the fan is off

Heating up…
Stopped print with XYZ blinking…

### CODE REVIEW

File: Marlin/dogm_lcd_implementation.h [hunk 1]
```diff
@@ -279,26 +279,32 @@ static void lcd_implementation_init() {
 
 static void lcd_implementation_clear() { } // Automatically cleared by Picture Loop
 
+static void _draw_centered_temp(int temp, int x, int y) {
+  int degsize = 6 * (temp >= 100 ? 3 : temp >= 10 ? 2 : 1); // number's pixel width
+  u8g.setPrintPos(x - (18 - degsize) / 2, y); // move left if shorter
+  lcd_print(itostr3(temp));
+  lcd_printPGM(PSTR(LCD_STR_DEGREE " "));
+}
+
```
Comment (Blue-Marlin) [inline Marlin/dogm_lcd_implementation.h]: Be careful with calculations here. This is time critical. 3 heaters \* ( is + target) \* stripes -> 12 for RRDFGD (ST7920) or 48 for the others.

**Comment by Blue-Marlin** [comment]
Congratulations.
If shifting up/down anything in the lower half, always check with language cn.

**Comment by Grogyan** [comment]
From an aesthetic point of view, i'd prefer to have the X, Y and Z still
displayed when there is a value to them other than zero

**Comment by Blue-Marlin** [comment]
@Grogyan 
Looking at the code he made the foto with shut down steppers during a blink-off phases.

File: Marlin/dogm_lcd_implementation.h (hunk 1 above)
Comment (thinkyhead) [inline Marlin/dogm_lcd_implementation.h]: Better to have this `FORCE_INLINE`? I bet we can find a few other places to chop down cycles too, yet still maintain a nice aesthetic.

**Comment by thinkyhead** [comment]
> From an aesthetic point of view, i'd prefer to have the X, Y and Z still
> displayed when there is a value to them other than zero

@Grogyan Sorry about the confusion. You are seeing the blinking text during the "off" blink there.

> If shifting up/down anything in the lower half, always check with language cn.

@Blue-Marlin Nothing is shifting up or down, so no problem there.

**Comment by Grogyan** [comment]
😅
Great timing with that photo.

**Pull request merged**